.github/workflows/macM1-e2e.yaml

```diff
@@ -0,0 +1,50 @@
+name: e2e-MacM1
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '0 9 * * 1-5'
+
+
+jobs:
+
+  e2e-tests:
+    timeout-minutes: 45
+    runs-on: ProvoMacM1
+    env:
+      M1: 1
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+          ref: main
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '^1.18'
+      - name: Install dependencies
+        run: npm ci
+      - name: Post install script
+        run: npm run postinstall
+      - name: add tools to PATH
+        run: |
+          mkdir -p ~/.rd/bin
+          cp -R ~/actions-runner/_work/rancher-desktop/rancher-desktop/resources/darwin/bin ~/.rd/bin
+          export PATH="~/.rd/bin:$PATH"
+      - name: Run Rancher Desktop in dev
+        run: |
+          npm run dev --no-modal-dialogs &
+          sleep 300
+          ~/.rd/bin/rdctl shutdown
+          wait
```

Comment:
Why is this necessary? Please add a comment (in the workflow file) so we know in the future.

Note that the existing Cirrus CI tests don't do this, so it _shouldn't_ be necessary.

Reply:
This is part of the workaround of #2915.